src/managers/input/InputManager.cpp

```diff
@@ -776,6 +776,7 @@ void CInputManager::onMouseWheel(IPointer::SAxisEvent e) {
         const auto PWINDOW     = g_pCompositor->vectorToWindowUnified(MOUSECOORDS, RESERVED_EXTENTS | INPUT_EXTENTS | ALLOW_FLOATING);
 
         if (PWINDOW) {
+            factor *= PWINDOW->getScrollFactor();
```

Comment:
This multiplies the compositor-wide factor instead of overriding it entirely. This means that if I have the global factor set to `0.1`, I'd have to set the window scrollfactor to 10 to get a final factor of 1.

This might be correct, but I'm unsure.

Reply:
That's correct.

Originally, I was going to use the override approach, but there are two different scroll factors, one for mouse and one for touchpad, and if I were to add a window rule that overrides one, then I would feel obligated to add a second one that overrides the other.

This approach allows it to be either or.

Reply:
Sure, this makes sense. But doesn't that still pose the problem that the touchpad/mouse scroll factors are different, this also affects them differently?

A concrete example I have: global touchpad scroll factor is 0.1 due to GTK3 having a way too fast scroll. foot is not affected and could use a scroll factor of 1. This means if I set the scrollfactor windowrule to 10, it would also affect mouse scrolling, which I don't want it to.

Reply:
Alright, you've convinced me. I will use the override approach and add a window rule for touchpad scroll factor.